packages/sdk/src/utils/number.ts

```diff
@@ -177,9 +177,17 @@ export const formatCurrency = (
 export const formatDollars = (value: WeiSource, options?: FormatCurrencyOptions) =>
 	formatCurrency('sUSD', value, { sign: '$', ...options })
 
-export const formatPercent = (value: WeiSource, options?: { minDecimals: number }) => {
-	const decimals = options?.minDecimals ?? 2
-
+export const formatPercent = (
+	value: WeiSource,
+	options?: { minDecimals?: number; suggestDecimals?: boolean; maxDecimals?: number }
+) => {
+	let decimals = suggestedDecimals ? suggestedDecimals(value) : options?.minDecimals ?? 2
```

Comment:
I think this should be `let decimals = options?.suggestDecimals ? suggestedDecimals(value) : options?.minDecimals ?? 2`.